include/seqan3/alignment/configuration/align_config_gap_cost_affine.hpp

```diff
@@ -0,0 +1,112 @@
+ * \author Wiep van der Toorn <w.vandertoorn AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <seqan3/alignment/configuration/detail.hpp>
+#include <seqan3/core/algorithm/pipeable_config_element.hpp>
+#include <seqan3/core/detail/strong_type.hpp>
+
+namespace seqan3::align_cfg
+{
+// ------------------------------------------------------------------
+// seqan3::align_cfg::open_score
+// ------------------------------------------------------------------
+
+/*!\brief A strong type of underlying type `int32_t` that represents a score (usually negative) that
+ *        is incurred once per stretch of consecutive gaps.
+ * \ingroup scoring
+ * \see seqan3::align_cfg::gap_cost_affine
+ */
+struct open_score : seqan3::detail::strong_type<int32_t, open_score, seqan3::detail::strong_type_skill::convert>
+{
+   //!\brief The type of the strong type base class.
+   using base_t = seqan3::detail::strong_type<int32_t, open_score, seqan3::detail::strong_type_skill::convert>;
+   using base_t::base_t; // Import the base class constructors
+};
+
+ // ------------------------------------------------------------------
+ // seqan3::align_cfg::extension_score
+ // ------------------------------------------------------------------
+
+ /*!\brief A strong type of underlying type `int32_t` that represents the score (usually negative) of any character
+  *        against a gap character.
+  * \ingroup scoring
+  * \see seqan3::align_cfg::gap_cost_affine
+  */
+ struct extension_score : seqan3::detail::strong_type<int32_t,
+                                                      extension_score,
+                                                      seqan3::detail::strong_type_skill::convert>
+ {
+     //!\brief The type of the strong type base class.
+     using base_t = seqan3::detail::strong_type<int32_t, extension_score, seqan3::detail::strong_type_skill::convert>;
+     using base_t::base_t; // Import the base class constructors
+ };
+
+/*!\brief A configuration element for the affine gap cost scheme.
+ * \ingroup alignment_configuration
+ *
+ * \details
+ *
+ * Configures the gap scheme for the alignment algorithm. The gap scheme determines how gaps are penalised inside
+ * of the alignment algorithm. If the gap scheme is not configured, it will default to a linear gap scheme initialised
+ * with edit distance. Note that the gap open score is used as an additional score. This means that the score for
+ * opening a gap during the affine alignment execution is the sum of the gap score and the gap open score.
+ *
+ * ### Example
+ *
+ * \include test/snippet/alignment/configuration/align_cfg_gap_cost_affine_example.cpp
+ *
+ */
+struct gap_cost_affine : public pipeable_config_element<gap_cost_affine>
+{
+public:
+    //!\brief The score per consecutive sequence of gaps. Defaults to 0.
+    seqan3::align_cfg::open_score open_score{0};
+    //!\brief The cost per gap character. Defaults to -1.
+    seqan3::align_cfg::extension_score extension_score{-1};
```

Comment:
```suggestion
    int32_t open_score{0};
    //!\brief The cost per gap character. Defaults to -1.
    int32_t extension_score{-1};
```

I meant to change it like this :)